lib/model.js

```diff
@@ -2022,6 +2022,8 @@ class Model {
   static create(values, options) {
     options = Utils.cloneDeep(options || {});
 
+    this._injectScope(options);
```

Comment:
scopes should be injected into something which accepts `options.where`, there is no point of using scopes in create/bulkCreate methods

Reply:
Understood.  I'll close this.